src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -163,6 +163,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <NETSdkWarning Condition="'$(PublishTrimmed)' == 'true' and '$(_TargetFrameworkVersionWithoutV)' &lt; '3.0'"
                    ResourceName="PublishTrimmedRequiresVersion30" />
 
+    <!-- Generate Trimming warnings for WinForms and Wpf applications-->
+    <NetSdkWarning Condition="'$(UseWindowsForms)' == 'true' and '$(PublishTrimmed)' == 'true'"
```

Comment:
We wanted to make this an error, right? And make it conditional on an additional internal property, `_SuppressWindowsFormsTrimError` and `_SuppressWpfTrimError`?

Reply:
I was not sure if we decided definitely on the error/warning option (we've gone back and forth in the earlier discussion on this). 

Do we need an additional property to suppress this when the actual warning can be suppressed (NoWarn: NETSDK1168) if we go with the warning?

This behavior seems a reasonable compromise to me,

- We clearly indicate our recommendation that trimming is not compatible (warning and a hard block with built-in COM disabled) 
- Allow developers who still want to go ahead, to suppress the warning and information to turn on feature switches via our docs (after again stating other options recommending to use `COMWrappers `instead of built-in COM in our docs). Making this option too painful by erroring out seems overly cautious to me given we protect developers who are unaware of trimming problems with our default option above.

Reply:
I agree we don't need an extra property if it's just a warning. Let's at least move forward with the warning since that's the high-order bit.